build/workbox.config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "globDirectory": "./",
+    "globPatterns": [
+        "assets/**/*.{js,html,svg,png,css,jpg,ico}",
+        "dist/**/*.{js,html,svg,png,css,jpg,ico}"
+    ],
+    "swSrc": "sw-src.js",
```

Comment:
How about just naming this `sw.js`? We output the generated file in the _gh_pages folder so we don't have any conflicts.